examples/moon.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="https://aframe.io/releases/0.3.0/aframe.min.js"></script>
+<script src="../index.js"></script>
+</head>
+<body>
+<h1>This goes here!</h1>
```

Comment:
I use it as page filler, so I can tell if the 2.5d version is staying behaved and not taking over pages. I don't mind getting rid of it, and letting the app be fullscreen.